_pages/en_US/troubleshooting.txt

```diff
@@ -564,10 +550,16 @@ ARM11 exception handlers are disabled, or custom firmware is not installed. Try
 <details>{{ compat | markdownify }}</details>
 
 {% capture compat %}
-<summary><u>Exception screen when booting/loading an application</u></summary>
+<summary><u>My device does not show any of my installed apps on the HOME Menu</u></summary>
 
-Look for your exception screen in [this page](https://wiki.hacks.guide/wiki/3DS:Error_screens/Luma3DS_exception_screen).
-If you weren't able to find your error or the instructions didn't work, join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) for further assistance.
+This could be caused by various reasons, but mostly because your SD card is not being read by the system.
+You can check if your SD is not being read by holding SELECT on boot and checking the yellow text on the bottom screen; if it says "Booted from CTRNAND via B9S", then your system is booting from the internal memory and not from the SD card.
```

Comment:
not being read -> being read (negation doesn't particularly matter but it flows better when not negated)